tutorials/SequenceAlignment.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Structural alignment of proteins by sequence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the field of Bioinformatics is usual to perform a BLAST search in order to find\n",
+    "similar proteins to the one of your interest. If the structures of some of the results\n",
+    "are available, you might want to align them all together to see the differences. \n",
+    "These structures might not have the same number of residues, or the exact same residues,\n",
+    "or even the labeling of the atoms can be different, which makes a typical alignment function fail.\n",
+    "However, HTMD provides the function ```htmd.molecule.util.sequenceStructureAlignment```, which takes two proteins and\n",
```

Comment:
Don't use However here. It makes it sound a bit like we did something cooler than this is haha. Maybe "therefore" is better.